### DISCUSSION TITLE
Please answer my support ticket. I have waited for 2 weeks and our company has paid account.

### DISCUSSION THREAD

**Discussion by flywhc** [community/community#62112] [Copilot Conversations]
My ticket is #2237338
Our company paid for the service but I am still unable to use the copilot account after many weeks.
I thought answering the support question should be part of the service. Do you have other support channels for paid users?

My question was:
Our IT admin asked me to get VS benefit to solve the error 'our administrator has configured the application sso_IAM_GitHub.com to block users'.
Does joining a GitHub organization with Copilot account requires a 'Visual Studio subscription benefit - GitHub enterprise'?

**Comment by GhostOf0days** [comment]
Could you please try [recreating the ticket](https://support.github.com/)? Also, please remove your ticket number from the post. It could pose a risk to you.

**Reply by queenofcorgis** [reply]
@GhostOf0days - There is no risk to a user posting their ticket number on the community, in the future please do not encourage submitting multiple support tickets.

**Comment by queenofcorgis** [comment]
Hi @flywhc, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident. 

For more details on what your GitHub Copilot subscription includes, check out our [website](https://github.com/features/copilot/). 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. 

Thank you!
